rocketpy/rocket/rocket.py

```diff
@@ -714,20 +714,20 @@ def evaluate_inertias(self):
         """
         # Get masses
         prop_mass = self.motor.propellant_mass  # Propellant mass as a function of time
-        dry_mass = self.dry_mass  # Constant rocket mass with motor, without propellant
 
         # Compute axes distances
-        CM_to_CDM = self.center_of_mass - self.center_of_dry_mass_position
-        CM_to_CPM = self.center_of_mass - self.center_of_propellant_position
+        CDM_to_CPM = (
+            self.center_of_dry_mass_position - self.center_of_propellant_position
+        )
 
         # Compute inertias
-        self.I_11 = parallel_axis_theorem_from_com(
-            self.dry_I_11, dry_mass, CM_to_CDM
-        ) + parallel_axis_theorem_from_com(self.motor.I_11, prop_mass, CM_to_CPM)
+        self.I_11 = self.dry_I_11 + parallel_axis_theorem_from_com(
+            self.motor.I_11, prop_mass, CDM_to_CPM
+        )
```

Comment:
Should we really use the propellant mass in here? Or the motor total mass?

If we use the prop_mass, than I think we should use `self.motor.propellant_I_11` instead of `self.motor.I_11`.

Does it make enough sense to you?

```suggestion
        self.I_11 = self.dry_I_11 + parallel_axis_theorem_from_com(
            self.motor.I_11, *****prop_mass*****, CDM_to_CPM
        )
```

Reply:
Same problem happens in other parts in this code.

Reply:
You are totally right. Good catch